tests/bootstrap.php

```diff
@@ -4,11 +4,11 @@
 	die( 'Not an entry point' );
 }
 
-error_reporting( E_ALL | E_STRICT );
-ini_set( 'display_errors', 1 );
+error_reporting( -1 );
```

Comment:
What was wrong with `E_ALL | E_STRICT`?

Reply:
Does not always include everything. If a new option gets added it will be missing

Reply:
How? E_ALL is everything.

Reply:
It is not. If it was, why would you ever do `E_ALL | E_STRICT` in the first place?

Reply:
FWIW: http://php.net/manual/en/function.error-reporting.php says that since PHP 5.4:
> E_STRICT became part of E_ALL

so maybe that was the source of the confusion.

That said, the same page has a tip: 
> Passing in the value -1 will show every possible error, even when new levels and constants are added in future PHP versions. The E_ALL constant also behaves this way as of PHP 5.4. 

which I in my eyes makes Jeroen's change the neatest solution to have.

Reply:
"Neat" how? E_ALL is identical to -1. E_ALL includes everything. I know this was not the case for some PHP versions, namely 5.0 to 5.3. But we don't care about these any more. I know using -1 is kind of a "tradition". But it's cryptic, and doesn't need to be cryptic any more.